R/dependencies.R

```diff
@@ -0,0 +1,112 @@
+##' @title Get the dependencies for a given report from the orderly_run.yml
+##' 
+##' @param report the name of the report
+##' @param id the id of the report, if omitted, use the id of the latest report
+##' @param config_path Path to the orderly configuration
+##' @param draft If true look in the draft directory otehrwise look in archive
+##'
+##' @export
+get_dependencies <- function(name, id = NULL, config_path = NULL, draft = FALSE) {
```

Comment:
All the functions here need changing in name I think - take a look at [orderly's `NAMESPACE`](https://github.com/vimc/orderly/blob/master/NAMESPACE) for what we currently export.  I think that better names might be `orderly_dependencies` here perhaps

Reply:
Also I think for consistency this should include `locate` as per other other functions, the you can use `inheritParams` like https://github.com/vimc/orderly/blob/21845fde4a1dc73cc5fa27c52a800c59446fc8a6/R/query.R#L25-L34